.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,7 @@
 ## Checklist
 
 - [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
+- [ ] Signed of each commit with a [Developer Certificate of Origin (DCO)](https://probot.github.io/apps/dco/) 
```

Comment:
Unfortunately "sign commit" and "sign-off commit" are two totally different things... 

- Sign https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits (`-S`, capital S), uses GPG, SSH, or S/MIME
- Signoff https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository (`-s`, lowercase s)

The former is much more common and the GH interface has native support for it. For signoffs though it's trickier and regular people have no idea what that means I fear.